### PR TITLE
IAMuser

### PR DESCRIPTION
**Pull request by miifeanyi**

### CODE REVIEW

File: s3.tf [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    key    = "project/terraform.tfstate"
+    region = "us-east-1"
+  }
+}
+
+module "s3_bucket" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+
+  bucket = "eks.bucket"
+  acl    = "public-read-write"
```
Comment (vladimirmukhin) [inline s3.tf]: this setting makes all resources in your bucket public, please disable it

Comment (miifeanyi) [reply s3.tf]: updated acl value to private

File: remote-state/dynamostatelock.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
```
Comment (vladimirmukhin) [inline remote-state/dynamostatelock.tf]: 'basic-dynamodb-table' is not the best name. try choosing more meaningful name for this resource

File: remote-state/dynamostatelock.tf [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  billing_mode = "PAY_PER_REQUEST"
+  hash_key     = "LockID"
+
+  attribute {
+    name = "LockID"
+    type = "S"
+  }
+
+  tags = {
+    Name        = "dynamodb-table-1"
```
Comment (vladimirmukhin) [inline remote-state/dynamostatelock.tf]: Name tag is different from name property

**Review by vladimirmukhin — APPROVED** [review]

File: remote-state/dynamostatelock.tf (hunk 2 above)
Comment (miifeanyi) [reply remote-state/dynamostatelock.tf]: changed resouce name

File: remote-state/dynamostatelock.tf (hunk 3 above)
Comment (miifeanyi) [reply remote-state/dynamostatelock.tf]: change made to name tag

**Pull request merged**